Examples/Swift/ManageOfflineRegionsExample.swift

```diff
@@ -1,220 +1,234 @@
-                print(Float((packArray[indexPath.row].progress.countOfResourcesCompleted)/(packArray[indexPath.row].progress.countOfResourcesExpected) * 100))
-
-                cell.textLabel?.font = UIFont.systemFont(ofSize: 15)
-                cell.textLabel?.numberOfLines = 0
-                cell.textLabel?.lineBreakMode = .byWordWrapping
-                if #available(iOS 13.0, *) {
-                    cell.textLabel?.textColor = .darkGray
-                } else {
-                    // Fallback on earlier versions
-                }
-                cell.textLabel?.textAlignment = .left
+    deinit {
+        // Remove offline pack observers.
+        print("Removing offline pack notification observers")
+        NotificationCenter.default.removeObserver(self)
 
-            } else {
+    }
 
-                cell.textLabel?.text = "No Offline Packs Saved"
-                cell.textLabel?.textColor = .darkGray
-                cell.textLabel?.textAlignment = .center
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
 
-            }
+        if let packs = MGLOfflineStorage.shared.packs {
+            return packs.count
+        } else {
+            return 1
+        }
+    }
 
-            return cell
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        var cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath)
+        cell = UITableViewCell(style: .subtitle, reuseIdentifier: "cell")
+        if let packs = MGLOfflineStorage.shared.packs {
+            let completed = Float(packs[indexPath.row].progress.countOfResourcesCompleted)
+                let expected = Float(packs[indexPath.row].progress.countOfResourcesExpected)
+                let progress = completed / expected
+            cell.textLabel?.text = "Region \(indexPath.row + 1): size: \(packs[indexPath.row].progress.countOfBytesCompleted)"
+            cell.detailTextLabel?.text = "Percent completion: \(String(format: "%.2f", progress * 100))%"
+            cell.textLabel?.textAlignment = .left
+
+        } else {
+            print("so...")
+            cell.textLabel?.text = "No Offline Packs Saved"
+            cell.textLabel?.textColor = .darkGray
+            cell.textLabel?.textAlignment = .center
 
         }
 
-        func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-            let selectedRegion = regionArray[indexPath.row]
+        return cell
+
+    }
+
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        guard let packs = MGLOfflineStorage.shared.packs else { return }
+        if let selectedRegion = packs[indexPath.row].region as? MGLTilePyramidOfflineRegion {
             mapView.setVisibleCoordinateBounds(selectedRegion.bounds, animated: true)
         }
+    }
 
-        @objc func startOfflinePackDownload() {
+    @available(iOS 11.0, *)
```

Comment:
I didn't realize this was an iOS 11+ things. Let's leave this part out since we don't want to have to deal with many `@available` checks in examples, as its not integral to the example being effective.